### ISSUE TITLE
use check_date:existence instead of check_date in the existence quest

### ISSUE THREAD

**Issue by Lukas458** [streetcomplete/StreetComplete#2508]
**Use case**
Hello,
at the moment, for the existence quest there is used the key **check_date**, to indicate when the existence of a feature was surveyed last time. 
The osm-wiki says to check_date: *The tag is intended to document the last date, when an object as a whole or specific tags of the object were checked and verified. [...] To indicate the review of specific tags the key is built from multiple words separated by colons like check_date:collection_times*

The thing is now, that by the existence quest, *only* the existence of a feature is surveyed, but no other tags (as far as I know).
So being exactly, the check_date is related only to the existence, like check_date:collection_times is related only to the collection_times.

It might be, that check_date:existence is not so common until now. Some users may also use just *check_date* in their changes when only the existence of a feature has been surveyed, too. But strictly speaking, a general *check_date* means that *all* tags of a feature have been checked! If it would not, we wouldn't need check_date:collection_times and others.

Now let me give you three examples, why a differentation between a general check_date and check_date:existence is needed. Please note that all these examples are osm-objects, where some modification of tags is needed, but I've left it out until now to show their history with the existence quest:

https://www.openstreetmap.org/node/2141756933/history
It was asked after the existence of the phone and check_date was added.
This is okay, but there's a tag which is outdated. covered=booth is not true anymore. Since end of 2020, it's covered=no.

https://www.openstreetmap.org/node/4039181173/history
It was asked after the existence of the phone and check_date was added.
But there are some outdated tags. The ref:number_of_position is outdated, but okay, this is a special key. But also, the payment methods have changed. The support for credit cards and "ep_geldkarte" has been removed by the phone's operator. **But the check_date from 2021 would suggest that everything is okay and current.**

https://www.openstreetmap.org/node/695686332
It was asked after the existence of the phone and check_date was added.
But there's a tag which was wrong since version 1 of the node's existence, because the value for the operator is not correct.
**But check_date suggets that all tags would have been checked in 2021.**

Of course, people using SC cannot know these details, because they were asked only about the existence of the objects. Also, it's only one class of objects, but I think the claim is justified nevertheless, that check_date:existence would be more exactly. It would have to be cleared what would happen then with a general check_date, if it has been set by another user before.

**Proposed Solution**
As it would not be possible for SC to check out all tags of every object which is affected by the existence quest (truely, the existence quest is also not supposed for that), my suggestion would be to use **check_date:existence** instead of **check_date** in the existence quest. 
For existing check_dates there could be the rule that SC recognizes them (as it is already) for looking how often there has to be asked about the object at all, but not overwrites them, just let them be, because for me, a check_date means that all existing tags have been checked at this time. check_date:existence could then be added, beacuse it sure can be, that the existence was checked a few days ago, but that all tags have been checked could be more time ago. So, check_date and check_date:existence for me could exist also both with different values.

**Comment by westnordost** [comment]
`check_date:existence` is a new tag that is not (really) used (yet). So, you should post this rather to an OpenStreetMap channel than here.

**Comment by Lukas458** [comment]
Hmm, but wasn't cycleway:both not in use before SC began to tag it, too?

**Comment by Lukas458** [comment]
The wiki page says that check_date is a key which can be expanded for the tag you want to affect.

**Comment by westnordost** [comment]
But `existence` is not a tag. Also, this is about compatibility with other data users. `check_date` is well known and more likely to be supported than if I make up something new.

**Comment by westnordost** [comment]
In the original #2074 issue I mentioned maybe `check_date:existence` could be used and I thought I had mentioned why I decided to use `check_date`. Reading it again, apparently I did not mention it. Summary:
1. `check_date:existence` is a new tag that is not used yet, thus not understood by other data users. New tags should not be introduced by StreetComplete unless discussed with the broader community first
2. `check_date:existence` collides with the `check_date:[name of key]` schema. This could be an issue for data users that process/edit `check_date:[name of key]` in the same way `recycling:check_date=2000-10-10` or `recycling:note=lorem ispum dolor est` could be a problem. For example imagine an editor would "clean up" check dates for keys that don't exist anymore.
3. `check_date` is already documented ambiguously enough to allow it to be used to just have the existence checked, i.e. current usage by StreetComplete is not against intended usage of that tag
4. The claim that `check_date` should only be set if "everything" has been checked is unrealistic because it can theoretically contain any number of things. Most importantly, it will often consist of a mixture of information, some of it is verifiable on the ground, some of it is verifiable via other means (satellite imagery, looking on the website of a business, ...), some is not verifiable without special tools, some is only verifiable with special knowledge, some may not be verifiable at all. For example 
```amenity=bicycle_parking
name=Stadtradstation Stadthaus
ref=5551-234
phone=555-4567913
operator=Stadtverkehrsverbund Stadthausen
svv:urksburgs:specialtag=6543-a
note:wheelchair=Toiletten im 1OG, Rampe für Fahrräder etwas steil. 1 OG nur offen im Sommer
ele=14.3
```
So in other words, since it is unrealistic to expect that someone is able (either as a surveyor or "on the couch") to check "all the things" at the same time, it is unrealistic to even assume that `check_date` is used or should be used that way. So, de-facto situation is that `check_date` means "someone was there and checked _something_ here, at least its existence".

Doesn't mean that something like `check_date:existence` would be a bad idea, just describing my reasons why I did use  `check_date` for the "is X still here?" quest.

**Comment by westnordost** [comment]
So, I'll close this for now. There is always the option to discuss this with the broader OSM community. If you do, maybe bring up the arguments I did in my lat comment as well.

**Comment by mnalis** [comment]
While I agree that `check_date:existence` is [wrong](https://wiki.openstreetmap.org/wiki/Talk:Key:check_date) (as there is no tag `existence` on which `check_date` would operate) and not used according to taginfo, I note that [Key:check_date](https://wiki.openstreetmap.org/wiki/Key:check_date#Other_tags_in_use) wiki does mention related tag `checked_exists:date` with about [610 uses](https://taginfo.openstreetmap.org/keys/?key=checked_exists%3Adate), which does sound like what is wanted here (I've reformatted, cleaned up and updated the wiki there a little to make it more understandable).

**Comment by Lukas458** [comment]
Thank you @mnalis . The key you mentioned and the wiki page sounds/looks well. But maybe my understanding of check_date was a bit strict, yes. It depends on what kind of object it is, concerning how difficult it is to check all tags which are set. The option to discuss this with the OSM community is there, yes. At the moment I'm not sure, but this would have to be done surely before something happens here, yes.
